wire/msgversion.go

```diff
@@ -18,7 +18,7 @@ import (
 const MaxUserAgentLen = 2000
 
 // DefaultUserAgent for wire in the stack
-const DefaultUserAgent = "/btcwire:0.2.1/"
+const DefaultUserAgent = "/btcwire:0.2.2/"
```

Comment:
This should probably be  `0.3.0`.  Typically it's `<major>.<minor>.<patch>` and this is adding a new flag and bumping the protocol version, so it's more than a patch.